internal/common/labels.go

```diff
@@ -37,7 +41,7 @@ func AddAppLabels(requestInstance *v1beta1.SSP, name string, component AppCompon
 
 	labels[AppKubernetesNameLabel] = name
 	labels[AppKubernetesComponentLabel] = component.String()
-	labels[AppKubernetesManagedByLabel] = "ssp-operator"
```

Comment:
You will have to add here the `AppKubernetesManagedByValue = "tekton-tasks-operator"` variable too, because during upgrade, ssp operator will have to look for objects with `app.kubernetes.io/managed-by: tekton-tasks-operator` label and take ownership of them

Reply:
Okay, but don't we have only one key (`AppKubernetesManagedByValue`) to use? Can we have multiple same keys?

Reply:
you can have only one key, but I mean it like:
Objects which already exists in cluster, have TTO's `AppKubernetesManagedByValue` label. During upgrade ssp operator will have to look for all objects with  TTO's `AppKubernetesManagedByValue` label and change to label to SSP value